core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -82,7 +95,7 @@ var TopicsAndSkillsDashboardPage = function() {
     var topicsListElems = [];
     var topicsListItemsCount = await topicsListItems.count();
     for (var i = 0; i < topicsListItemsCount; i++) {
-      var topic = await topicsListItems.get(i);
+      var topic = topicsListItems.get(i);
```

Comment:
Just checking, why delete the "await"? Ditto elsewhere. /cc @U8NWXD

Reply:
You can take a look at [this](https://github.com/oppia/oppia/pull/9333#discussion_r433345229).

Reply:
Yep, `.get()` returns an ElementFinder synchronously, and then operations on that ElementFinder are asynchronous